lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -131,7 +131,7 @@ prompt-toolkit==3.0.19; python_full_version >= "3.6.1" and python_version >= "3.
 protobuf==3.19.1; python_version >= "3.6"
 prov==1.5.1; python_version >= "3.6" and python_version < "4"
 psutil==5.8.0; (python_version >= "2.6" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0")
-pulsar-galaxy-lib==0.14.10
+pulsar-galaxy-lib==0.14.12
```

Comment:
You need to also update this in `lib/galaxy/dependencies/dev-requirements.txt`

Reply:
Oppps - thanks!